src/pykiso/lib/connectors/cc_pcan_can/cc_pcan_can.py

```diff
@@ -540,6 +547,9 @@ def stop_pcan_trace(self):
         Stops the PCAN trace if it is currently running.
         :return: None
         """
+        if not self.logging_activated:
+            log.warning("Logging is not activated")
```

Comment:
Why is this a warning?

Reply:
It was because the user could call the function.